### PR TITLE
[WIP] Merge master into bg/peps-ideas

### PR DESCRIPTION
**Pull request by lpawela**

### CODE REVIEW

File: src/ising.jl [hunk 1]
```diff
@@ -133,7 +62,7 @@ Create the Ising spin glass model.
 
 Store extra information
 """
-function ising_graph(instance::Instance, L::Int, β::Number=1, sgn::Number=1)
+function ising_graph(instance::Instance, L::Int, β::Number=1, sgn::Number=-1)
```
Comment (lpawela) [inline src/ising.jl]: Kill it with fire!

**Comment by lpawela** [comment]
@kdomino 
```
LAPACKException(7)
...
construct_mps(::MetaGraph{Int64,Float64}, ::Float64, ::Int64, ::Int64, ::Float64) at /Users/runner/work/SpinGlassPEPS.jl/SpinGlassPEPS.jl/src/mps_implementation.jl:144
```
Seems random, sometimes the tests pass. Please check.

**Comment by kdomino** [comment]
It is random, most probably due to compression.

**Comment by lpawela** [comment]
> It is random, most probably due to compression.

@kdomino then we should enforce a random seed which guarantees that the tests pass. Tests MUST be deterministic.

**Comment by lpawela** [comment]
@kdomino 
One more remark. This exception usually happens when the matrices contain `Inf` or `NaN`. This probably shoud never happen.

**Comment by kdomino** [comment]
I realy do not know how to copy with this

**Comment by kdomino** [comment]
cope

**Comment by kdomino** [comment]
I enforces seeds earlier in tests, it is what I could do

**Pull request merged**